packages/webdriverio/src/index.ts

```diff
@@ -23,6 +25,20 @@ import type {
 } from 'axe-core';
 import type { Options, CallbackFunction, PartialResults } from './types';
 
+let axeCorePath = '';
+async function loadAxePath() {
```

Comment:
nit - we use a function `loadAxePath` here but not puppeteer, IMO we should keep it consistent

Reply:
We use a function here because we need to do async stuff at the top-level in order to initialize `axeCorePath`. And for some reason that still isn't allowed. puppeteer does this inside an async function so it isn't needed.